.github/workflows/integration-test-flex-suite.yaml

```diff
@@ -1,9 +1,14 @@
 name: Integration test CephFlexSuite
-on: [pull_request]
+on: [push, pull_request]
 
 jobs:
   TestCephFlexSuite:
+    if: ${{ github.event_name == 'pull_request' && github.ref != 'refs/heads/master' }}
     runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        kubernetes-versions : ['v1.15.12', 'v1.20.2']
```

Comment:
I'm thinking we can safely cut back on how many of the test suites run on multiple versions for PRs. Let's just have the flex, helm and smoke suite run on both versions, and just run the others on 1.20. Thanks!

Reply:
@travisn `upgrade, helm and smoke` on both, `flex` on `v1.15.x`, `mgr, multicluster` on `v1.20.x` right?

Reply:
One small suggestion... `helm` and `smoke` on both, flex on `v1.15.x`, `upgrade`, `mgr`, `multicluster` on `v1.20.x`